_data/api.yml

```diff
@@ -9,10 +9,11 @@ toc:
   path: /docs/reference/mqtt-api/
 - title: CoAP Device API
   path: /docs/reference/coap-api/
-- title: HTTP Device API
-  path: /docs/reference/http-api/
 - title: LWM2M Device API
   path: /docs/reference/lwm2m-api/
+  new: true
+- title: HTTP Device API
```

Comment:
You have changed the order of transports in the menu. Please revert this and keep only "new: true".